### ISSUE TITLE
SC2154: False positive for .sh.version in ksh

### ISSUE THREAD

**Issue by mcandre** [koalaman/shellcheck#789]
In ksh, `.sh.version` is a standard variable referring to the shell version. Could we add this to the list of known, pre-defined variables?

#### For bugs
- Rule Id (if any, e.g. SC1000): SC2154
- My shellcheck version (`shellcheck --version` or "online"): 0.3.7

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/usr/bin/env ksh
echo "${.sh.version}"
```

#### Here's what shellcheck currently says:

```
In ./lib/version line 2:
echo "${.sh.version}"
      ^-- SC2154: .sh.version is referenced but not assigned.
```

#### Here's what I wanted or expected to see:

(no warning output)

**Comment by koalaman** [comment]
7124c11 adds this to the list of known variables. Thanks!
